ansible_ai_connect/ai/api/model_pipelines/http/pipelines.py

```diff
@@ -144,7 +144,7 @@ def self_test(self) -> HealthCheckSummary:
             r = requests.get(
                 self.config.inference_url + "/readiness",
                 headers=headers,
-                timeout=self.timeout(1),
+                timeout=1,
```

Comment:
I'd still like to be able to set the `timeout` for chat requests.

`ANSIBLE_AI_MODEL_MESH_CONFIG` supports configuring a `timeout` for the "Chatbot" pipelines.

This change negates the configuration option.

Reply:
Thank you for catching this. It's definitely a mistake.

Reply:
Oh crap, indeed. I just pushed an update, stay with `task_gen_timeout()` for now.